test/fixtures/compare-im-references.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
Yes, will remove! :+1: These are still interesting to visually see where the differences are but I am ok to remove it to reduce the size of the project.